etc/conf/server.properties

```diff
@@ -1,4 +1,10 @@
 server.env=dev
+server.authorization=disable
+server.authorization-url=
```

Comment:
Can you document the `authorization-url`, `token-url`, `client-id`, `client-secret` with an example?

Reply:
ok added some samples for Google